python/across_api/base/footprint.py

```diff
@@ -0,0 +1,43 @@
+# Copyright © 2023 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+
+
+from astropy import units as u  # type: ignore[import]
+from astropy.coordinates.representation import (  # type: ignore[import]
+    CartesianRepresentation,
+    UnitSphericalRepresentation,
+)
+from astropy.coordinates.matrix_utilities import rotation_matrix  # type: ignore[import]
+
+
+class Footprint:
+    polygon: list = []
+
+    def __init__(self, polygon: list) -> None:
+        self.polygon = polygon
```

Comment:
You could make this a dataclass and thereby remove the need for an explicit __init__ method.

Reply:
I kept the explicit `__init__` and instead of having `polygon` being the variable set, I just instantiated the representation field.